content/sql/concepts/commands/terms/union/union.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'UNION'
+Description: 'Allows the combination of the results of two or more SELECT queries.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'SQLite'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
+---
+
+The `UNION` command allows for the combination of the results of two or more [`SELECT`](https://www.codecademy.com/resources/docs/sql/commands/select) queries. When two queries with the same number of columns are joined together with the `UNION` command, the result is a single result that is a combination of the individual results of each query. `UNION` is placed between the two queries being combined.
```

Comment:
```suggestion
The `UNION` command allows for the combination of the results of two or more [`SELECT`](https://www.codecademy.com/resources/docs/sql/commands/select) queries. When two queries with the same number of columns are joined together with the `UNION` command, the result is a combination of the returned result from both queries. `UNION` is placed between the two queries being combined.
```
Seemed like this could be cleared up a bit. 'Results' seemed repeated too many times.